SECURITY.md

```diff
@@ -0,0 +1,71 @@
+
+
+## When to report a vulnerability
+
+*   When you think this project has a potential security vulnerability.
+*   When you suspect a potential vulnerability but you are unsure that it impacts this project.
+*   When you know of or suspect a potential vulnerability on another project that is used by this project.
+
+
+## Patch, Release, and Disclosure
+
+The maintainers will respond to vulnerability reports as follows:
+
+1. The maintainers will investigate the vulnerability and determine its effects and criticality.
+2. If the issue is not deemed to be a vulnerability, the maintainers will follow up with a detailed reason for rejection.
+3. The maintainers will initiate a conversation with the reporter within 3 business days.
+4. If a vulnerability is acknowledged and the timeline for a fix is determined, the maintainers will work on a plan to communicate with the appropriate community, including identifying mitigating steps that affected users can take to protect themselves until the fix is rolled out.
+5. The maintainers will also create a [Security Advisory](https://docs.github.com/en/code-security/repository-security-advisories/publishing-a-repository-security-advisory) using the [CVSS Calculator](https://www.first.org/cvss/calculator/3.0), if it is not created yet.  The maintainers make the final call on the calculated CVSS; it is better to move quickly than making the CVSS perfect. Issues may also be reported to [Mitre](https://cve.mitre.org/) using this [scoring calculator](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator). The draft advisory will initially be set to private.
+6. The maintainers will work on fixing the vulnerability and perform internal testing before preparing to roll out the fix.
+7. Once the fix is confirmed, the maintainers will patch the vulnerability in the next patch or minor release, and backport a patch release into all earlier supported releases.
+
+
+## Public Disclosure Process
+
+The maintainers publish the public advisory to this project's community via GitHub. In most cases, additional communication via Slack, Twitter, mailing lists, blog, and other channels will assist in educating the project's users and rolling out the patched release to affected users.
+
+The maintainers will also publish any mitigating steps users can take until the fix can be applied to their instances. This project's distributors will handle creating and publishing their own security advisories.
+
+
+## Confidentiality, integrity and availability
+
+We consider vulnerabilities leading to the compromise of data confidentiality, elevation of privilege, or integrity to be our highest priority concerns. Availability, in particular in areas relating to DoS and resource exhaustion, is also a serious security concern. The maintainer team takes all vulnerabilities, potential vulnerabilities, and suspected vulnerabilities seriously and will investigate them in an urgent and expeditious manner.
```

Comment:
This differs from the OpenLDAP's classification (e.g. OpenLDAP project doesn't class denial of service like a crash as a security issue).

Reply:
So, this is an okay statement for us to back up.  DDoS vulnerability is as much a "bug/security issue" as anything else.  I get that this is new, but is that not okay?

Reply:
DDoS **what**? Also don't see how you're going to persuade people to have e.g. an `assert()` failure expedited as a "security" bug. You'll get a lot of pushback from the project on that, and it might be considered passive-aggressive if you then point to this document. In those cases the project will immediately make such an issue public, and meeting any self-imposed deadlines to fix and manage the "vulnerability" admin will be on yourself.

You're within your rights to go ahead with that, just make sure that's what you really want.